scripts/e2e/bootstrap_job.template

```diff
@@ -9,6 +9,13 @@ spec:
       tolerations:
       - effect: NoSchedule
         key: node-role.kubernetes.io/master
+      initContainers:
+      - name: clusterctl-bin
```

Comment:
Can you explain what you're doing here?  What is this /tmp/clusterctl/clusterctl?